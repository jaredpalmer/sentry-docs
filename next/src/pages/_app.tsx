import * as React from "react";
import "../css/index.css";
import { AppProps } from "next/app";

function MyApp({ Component, pageProps }: AppProps) {
  return (
    <>
      <Component {...pageProps} />
    </>
  );
}

export default MyApp;
