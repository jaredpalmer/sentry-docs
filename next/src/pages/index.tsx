import Link from "next/link";
import React from "react";

export const Home = (): JSX.Element => (
  <div>
    <ul>
      <li>
        <Link href="/platforms/android/usage/advanced-usage/">
          <a>/platforms/android/usage/advanced-usage/</a>
        </Link>
      </li>
      <li>
        <Link href="/platforms/javascript/enriching-events/attachments/">
          <a>/platforms/javascript/enriching-events/attachments/</a>
        </Link>
      </li>
      <li>
        <Link href="/platforms/apple/usage/">
          <a>/platforms/apple/usage/</a>
        </Link>
      </li>
    </ul>

    <style jsx global>{`
      html,
      body {
        padding: 0;
        margin: 0;
        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
          Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
      }

      * {
        box-sizing: border-box;
      }
    `}</style>
  </div>
);

export default Home;
