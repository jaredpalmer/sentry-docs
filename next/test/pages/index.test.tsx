import React from "react";
import { Home } from "../../src/pages/index";
import { render } from "../testUtils";

describe("Home page", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<Home />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
