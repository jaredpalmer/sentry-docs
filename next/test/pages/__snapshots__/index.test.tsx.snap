// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="jsx-3892888070"
  >
    <ul
      class="jsx-3892888070"
    >
      <li
        class="jsx-3892888070"
      >
        <a
          class="jsx-3892888070"
          href="/platforms/android/usage/advanced-usage"
        >
          /platforms/android/usage/advanced-usage/
        </a>
      </li>
      <li
        class="jsx-3892888070"
      >
        <a
          class="jsx-3892888070"
          href="/platforms/javascript/enriching-events/attachments"
        >
          /platforms/javascript/enriching-events/attachments/
        </a>
      </li>
      <li
        class="jsx-3892888070"
      >
        <a
          class="jsx-3892888070"
          href="/platforms/apple/usage"
        >
          /platforms/apple/usage/
        </a>
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
